

numpy.exceptions.DTypePromotionError
====================================

.. currentmodule:: numpy.exceptions



.. autoexception:: numpy::exceptions.DTypePromotionError

